templates/primary-secondary/primary-secondary.js

```diff
@@ -46,88 +477,390 @@ class TemplatePrimarySecondary extends LitElement {
+			this._size = Math.max(desktopMinSize, desktopDividerSize * (1 / 3));
+		}
 	}
 
 	render() {
+		let tabindex;
+		const secondaryPanelStyles = {};
+		if (this._isResizable()) {
+			secondaryPanelStyles[this._isMobile ? 'height' : 'width'] = `${this._size}px`;
+			tabindex = 0;
+		}
+		const separatorMax = this._contentBounds && Math.round(this._isMobile ? this._contentBounds.maxHeight : this._contentBounds.maxWidth);
 		return html`
 			<div class="d2l-template-primary-secondary-container">
 				<header><slot name="header"></slot></header>
-				<div class="d2l-template-primary-secondary-content" data-background-shading="${this.backgroundShading}">
+				<div class="d2l-template-primary-secondary-content" data-background-shading="${this.backgroundShading}" ?data-animate-resize=${this._animateResize} ?data-is-collapsed=${this._isCollapsed} ?data-is-expanded=${this._isExpanded}>
 					<main><slot name="primary"></slot></main>
-					<div class="d2l-template-primary-secondary-divider"></div>
-					<aside><slot name="secondary"></slot></aside>
+					<div tabindex="${ifDefined(tabindex)}" class="d2l-template-primary-secondary-divider" role=separator  aria-orientation=${this._isMobile ? 'horizontal' : 'vertical'} aria-valuenow="${Math.round(this._size)}" aria-valuemax="${ifDefined(separatorMax)}">
+						<div class="d2l-template-primary-secondary-divider-handle" @click=${this._onHandleTap} @mousedown=${this._onHandleTapStart}>
+							<div class="d2l-template-primary-secondary-divider-handle-desktop">
+								<d2l-icon-custom size="tier1" class="d2l-template-primary-secondary-divider-handle-left">
+									<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg">
+										<path transform="rotate(90 9.004714965820312,9.000227928161623)" d="m13.708,6.29a1.006,1.006 0 0 0 -0.708,-0.29l-7.995,0a1,1 0 0 0 -0.705,1.71l4,4a1.013,1.013 0 0 0 1.42,0l4,-4a1.01,1.01 0 0 0 -0.013,-1.42l0.001,0z" fill="#494c4e"/>
+									</svg>
+								</d2l-icon-custom>
+								<div class="d2l-template-primary-secondary-divider-handle-line"></div>
+								<d2l-icon-custom size="tier1" class="d2l-template-primary-secondary-divider-handle-right">
+									<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg">
+										<path transform="rotate(-90 9.004714965820314,9.000227928161621)" d="m13.708,6.29a1.006,1.006 0 0 0 -0.708,-0.29l-7.995,0a1,1 0 0 0 -0.705,1.71l4,4a1.013,1.013 0 0 0 1.42,0l4,-4a1.01,1.01 0 0 0 -0.013,-1.42l0.001,0z" fill="#494c4e"/>
+									</svg>
+								</d2l-icon-custom>
+							</div>
+							<div class="d2l-template-primary-secondary-divider-handle-mobile">
+								${this._size === 0 ? html`<d2l-icon icon="tier1:chevron-up"></d2l-icon>` : html`<d2l-icon icon="tier1:chevron-down"></d2l-icon>`}
+							</div>
+						</div>
+					</div>
+					<div style=${styleMap(secondaryPanelStyles)} class="d2l-template-primary-secondary-secondary-container" @transitionend=${this._onTransitionEnd}>
+						<aside>
```

Comment:
Nit: could the `aside` and parent `div` be combined so it's just the `aside`?

Reply:
The `aside` is contained within a `div` so that the `aside` can maintain its width while the parent div collapses. This allows the secondary panel to be collapsed and expanded without losing scroll position. It also means that when the secondary panel collapses it looks like it slides off the page rather than resizing to a width of `0`.

Reply:
Ah cool makes sense!